SamcoEnhanced/OpenFIREFeedback.cpp

```diff
@@ -212,10 +221,12 @@ void FFB::RumbleActivation()
     if(rumbleHappening) {                                         // Are we in a rumble command rn?
         currentMillis = millis();                                 // Calibrate a timer to set how long we've been rumbling.
         if(SamcoPreferences::toggles.rumbleFF) {
-            if(currentMillis - previousMillisRumble >= SamcoPreferences::settings.rumbleInterval / 2) { // If we've been waiting long enough for this whole rumble command,
-                digitalWrite(SamcoPreferences::pins.oRumble, LOW);                         // Make sure the rumble is OFF.
-                rumbleHappening = false;                              // This rumble command is done now.
-                rumbleHappened = true;                                // And just to make sure, to prevent holding == repeat rumble commands.
+            if(!SamcoPreferences::toggles.autofireActive) {       // We only want to use the rumble timer is Autofire is not active. Otherwise, keep it going
```

Comment:
I think you meant "We only want to use the rumble timer **if** Autofire is not active. ..."